packages/react/src/components/variants-provider.component.tsx

```diff
@@ -29,6 +29,7 @@ const variantsScript = (variantsString: string, contentId: string) =>
     variants.forEach(function (template) {
       document.querySelector('template[data-template-variant-id="' + template.id + '"]').remove();
     });
+    document.getElementById('variants-script-${contentId}').remove();
```

Comment:
Is this just cleanup or is it solving an issue?

Reply:
solving an issue that happens only on prod builds, the script was being kept in ssr causing hydration issues in react 18